public/images/logos/wqxr-log-frq.svg

```diff
@@ -1,8 +1,10 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="220" height="34" viewBox="0 0 220 34">
+<a href="https://beta-wqxr.demo.nypr.digital/listen">
```

Comment:
this feels weird to me. can you embed an anchor link in a svg?

also, linking directly to `beta-wqxr` is going to require a code change when we go to production.